From: Etienne Millon <etienne.millon@gmail.com>
Date: Mon, 23 Apr 2012 22:11:22 +0200
Subject: Set visibility of get_next_word

---
 lib/stringlib.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/stringlib.c b/lib/stringlib.c
index ec4d367..f5f21ec 100644
--- a/lib/stringlib.c
+++ b/lib/stringlib.c
@@ -373,6 +373,7 @@ gchar * prepare_url(const gchar * URL, GlyrQuery * s, gboolean do_curl_escape)
 
 /* ------------------------------------------------------------- */
 
+__attribute__((visibility("default"))) // not sure about this one
 gchar * get_next_word(const gchar * string, const gchar * delim, gsize *offset, gsize len)
 {
     gchar * word = NULL;
